src/lib/contexts/NotificatonContext.tsx

```
@@ -0,0 +1,175 @@
+  | { type: "CLEAR_TOASTS" }
+
+const initialState: NotificationState = {
+  notifications: [],
+  unreadCount: 0,
+  toasts: [],
+}
+
+function notificationReducer(state: NotificationState, action: NotificationAction): NotificationState {
+  switch (action.type) {
+    case "ADD_NOTIFICATION":
+      const newNotifications = [action.payload, ...state.notifications]
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)